homeassistant/components/here_travel_time/coordinator.py

```diff
@@ -100,13 +98,7 @@ def _parse_routing_response(self, response) -> HERETravelTimeData:
         mapped_origin_lon: float = section["departure"]["place"]["location"]["lng"]
         mapped_destination_lat: float = section["arrival"]["place"]["location"]["lat"]
         mapped_destination_lon: float = section["arrival"]["place"]["location"]["lng"]
-        distance: float = summary["length"]
-        if self.config.units == IMPERIAL_UNITS:
-            # Convert to miles.
-            distance = DistanceConverter.convert(distance, LENGTH_METERS, LENGTH_MILES)
-        else:
-            # Convert to kilometers
-            distance = distance / 1000
+        distance: float = summary["length"] / 1000
```

Comment:
You could be more explicit here. Either keep the comment, or keep the convert:
```suggestion
        distance: float = DistanceConverter.convert(
            summary["length"], UnitOfLength.METERS, UnitOfLength.KILOMETERS
        )
```